client/components/Header/FraHeader/FraHeader.tsx

```diff
@@ -18,6 +19,7 @@ const FraHeader: React.FC = () => {
       <div className="app-header__menu">
         <LanguageSelector />
         <UserLinks />
+        <LinkGeo />
```

Comment:
why do we have LinkGeo twice ?

Reply:
It was in the fraHeader for desktop layout and countrySelect in the mobile layout. I moved it to countrySelect for both.